logical-comaprison-operators.js

```diff
@@ -47,33 +47,36 @@ const isHappy = false;
 
 // - Check if num is between 10 and 20 (inclusive) using the logical AND operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+console.log(num>10&&num<20);
 
 // - Check if num is either less than 5 or greater than 50 using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+console.log(num<5||num<50);
 
 // - Check if str is either "apple" or "orange" using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(str =="apple" || str =="orange");
 // - Check if isHappy value is true using the logical NOT operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(isHappy===true);
 // - Check if num is even and greater than 10 using the logical AND operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(num%2&&num>10);
 // - Check if num is divisible by both 3 and 5 using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(num/3||num/5);
 // - Check if str contains the letter "e". Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(str.search("e"));
 // - Check if str starts with "Hakuna". Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(str.startsWith("Hakuna"));
 // - Check if str ends with "a". Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(str.endsWith("a"));
 // - Check if num is either negative or odd using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(num<=-1||num%1);
```

Comment:
To check if the number is odd correctly you can do the following:
```
console.log( num % 2 === 1 )
```
Or 
```
console.log( num % 2 !== 0 )
```
either way these expression will return `true` or `false`